src/ui-kit/Select/Select.tsx

```diff
@@ -86,6 +91,36 @@ export const Select = JSX<SelectProps>(
   },
 );
 
+const renderLabel = (label?: string) =>
+  label ? (
+    <div className="mb-xs">
+      <Text size="text-m" color="text-primary-text" font="font-light">
+        {label}
+      </Text>
+    </div>
+  ) : null;
+
+const getFilteredOptions = (options: Option[], searchValue: string) => {
+  return searchValue
+    ? options.filter((_) => {
```

Comment:
опционально. у нас в двух местах уже используется следующая строковая почти-метрика:

```suggestion
    ? options.filter((_) => jaroWinklerDistance(_.text.toLowerCase())(searchValue.toLowerCase()) >= 0.8
```